internal/lm/nvml.go

```diff
@@ -218,6 +224,45 @@ func newGPUModeLabeler(devices []resource.Device) (Labeler, error) {
 	return labels, nil
 }
 
+func newImexDomainLabeler(devices []resource.Device) (Labeler, error) {
+	deviceTuple := map[string]string{
+		"clusterUuid": "",
+		"cliqueId":    "",
+	}
```

Comment:
Why a map instead of just two variables?

Reply:
done